client/components/jetpack/card/spp/jetpack-free-card-spp/index.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import React, { FC, useMemo } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import JetpackProductCardFeatures from 'calypso/components/jetpack/card/jetpack-product-card-i5/features';
+import JetpackFreeCardButton from 'calypso/components/jetpack/card/jetpack-free-card-button';
+
+/**
+ * Type dependencies
+ */
+import type { JetpackFreeProps } from 'calypso/my-sites/plans/jetpack-plans/types';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const JetpackFreeCardAlt: FC< JetpackFreeProps > = ( { siteId, urlQueryArgs } ) => {
+	const translate = useTranslate();
+
+	const freeFeatures = useMemo(
+		() => ( {
+			items: [
+				translate( 'Brute force attack protection' ),
+				translate( 'Site stats' ),
+				translate( 'Content delivery network' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Content Delivery Network' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/simplify-product-page-test)